plugins/plugin.xml

```diff
@@ -12,13 +12,14 @@
             </feature>
         </config-file>
         <source-file src="src/ios/OptimoveSDKPlugin.swift" />
+        <header-file src="src/ios/Bridging-Header.h" type="BridgingHeader" />
```

Comment:
We should update to add the InitializePluginManager source files (m & h)

Reply:
updated

Reply:
OK, should the .h file be a `<header-file>` entry?

Reply:
done